test/Graphics/VertexBuffer.test.cpp

```diff
@@ -90,6 +90,30 @@ TEST_CASE("[Graphics] sf::VertexBuffer", "[.display]")
         }
     }
 
+    SECTION("Move semantics")
+    {
+        SECTION("Construction")
+        {
+            sf::VertexBuffer       moveVertexBuffer(sf::PrimitiveType::LineStrip, sf::VertexBuffer::Dynamic);
```

Comment:
```suggestion
            sf::VertexBuffer       movedVertexBuffer(sf::PrimitiveType::LineStrip, sf::VertexBuffer::Dynamic);
```
Use a `moved` prefix instead of `move` to better match what is used elsewhere in the tests. Other files have this same inconsistency.